xc/common/cmake/device_define.cmake

```diff
@@ -336,6 +390,14 @@ function(ADD_XC_DEVICE_DEFINE)
         set(RR_PATCH_EXTRA_ARGS --synth_tiles ${SYNTH_TILES_LOCATION} ${RR_PATCH_EXTRA_ARGS})
     endif()
 
+    if(${USE_OVERLAY})
```

Comment:
Because USE_ROI and USE_OVERLAY are so closely related, could we fold the common elements?  I believe the only difference is the use of the `--overlay` flag?